backends/xnnpack/recipes/recipes.py

```diff
@@ -0,0 +1,51 @@
+# (c) Meta Platforms, Inc. and affiliates. Confidential and proprietary.
+
+# pyre-strict
+from typing import Any, Callable
+
+from executorch.backends.transforms.duplicate_dynamic_quant_chain import (
+    duplicate_dynamic_quant_chain_pass,
+    DuplicateDynamicQuantChainPass,
+)
+
+from executorch.backends.xnnpack.partition.xnnpack_partitioner import XnnpackPartitioner
+
+from executorch.backends.xnnpack.quantizer.xnnpack_quantizer import (
+    get_symmetric_quantization_config,
+    XNNPACKQuantizer,
+)
+from executorch.exir import ExportRecipe
+
+def get_generic_fp32_cpu_recipe() -> ExportRecipe:
```

Comment:
if namespaced to XNNPACK then cpu may not be needed?
```suggestion
def get_fp32_recipe() -> ExportRecipe:
```